net/softethervpn/Makefile

```diff
@@ -0,0 +1,113 @@
+#
+# Copyright (C) 2008-2016 OpenWrt.org
```

Comment:
Easy :) I'm not having a bad day and no objections to merging due to the above notice. The copyright issue just has been raised a while back and I am trying to make people aware of the fact that the copyright statement as-is is simply invalid and most people simply cargo-cult it.